doc/ant-xgboost_user_guide.md

```diff
@@ -0,0 +1,265 @@
+### _user guide:_ Ant-XGBoost on sqlflow
+
+#### Overview
+
+[Ant-XGBoost](https://github.com/alipay/ant-xgboost) is fork repo of [dmlc/xgboost](https://github.com/dmlc/xgboost), which is maintained by active contributors of dmlc/xgboost in Alipay.inc.
+
+Ant-XGBoost is built on top of dmlc/xgboost but with additional support on kubernetes and automatic training. 
```

Comment:
Use active voice instead of passive voice. We can say: Ant-XGBoost extends `dmlc/xgboost` with the capability of running on Kubernetes and automatic hyper-parameter estimation.